library/core/src/marker.rs

```diff
@@ -181,7 +181,7 @@ pub trait Unsize<T: ?Sized> {
 /// Required trait for constants used in pattern matches.
 ///
 /// Any type that derives `PartialEq` automatically implements this trait,
-/// *regardless* of whether its type-parameters implement `Eq`.
+/// *regardless* of whether its type-parameters implement `PartialEq`.
```

Comment:
I just noticed this typo in the docs and was considering proposing a fix, then I found this PR 🙂 which made me notice another one in the code (comments and message): `PartialStructuralEq` instead of `StructuralPartialEq` ([3 occurrences](https://github.com/search?q=repo%3Arust-lang%2Frust%20PartialStructuralEq&type=code)); is this an opportunity to correct that too?

Reply:
Ah, good catch, done.
(But it's not a done deal that this PR will land, so feel free to submit a separate one with just the typo fixes.)